wiki/Guides/osu!mania_Modding_Guide/en.md

```diff
@@ -0,0 +1,177 @@
+
+`00:52:299 - These pattern feels pretty weird while playing as there is a bias on the left hand, as well as a long anchor. You can solve that problem by moving 00:52:459 (52459|0) - this note to 3, 00:52:618 (52618|3) - this one to 2 and maybe remove 00:52:858 (52858|1) - because it causes a jack with the next hand pattern.`
+
+As seen, the format followed the guide above by first posting the timestamp and then describing the problem, finishing with a suggestion on how to fix it.
+
+Continue doing this process for the entire map. As mentioned earlier, if something is clearly unrankable, post it as problem instead of a suggestion so that the mapper sees they have to fix it. If suggesting pattern changes for several parts as shown above, do it as a suggestion.
+
+When giving a problem/suggestion towards a larger section of the beatmap, use two timestamps, one to start and one to end, and then describe the problem in this section, finishing with a suggestion on how to solve it. An example would be:
+
+`00:53:416 - to 00:58:682 - The usage of the 1st column here causes too many anchors which is really awkward to play alongside notes in the 2nd column. I'd suggest moving every other note (00:53:691 - , 00:53:966 - , 00:54:241 - , etc.) to a different column to alleviate the left hand bias in the map.`
+
+There are many ways how to mod a beatmap. The more practice given, the more experience earned and styles start to form with familiarity.
+
+### 2.7 Hitsound Modding
+Another important part of beatmaps are the hitsounds. If the beatmap has no hitsounds and the mapper still plans to add them, the best suggestion is to give them a friendly reminder that they are required for ranking a beatmap.
+
+If the beatmap does have hitsounds, feel free to mod them. The first thing to do is get an overview of the mappers idea on how the beatmap should be hitsounded. osu!mania has an advantage over other game modes as hitsound types can be displayed on the notes. To do so, go to “View” on the top left corner and then choose “Show Sample Name”. Now all used samples are shown on the notes.
+
+Next, **check if the hitsounds are audible enough**. Every beatmap must have audible hitsounds. Ideally, have the same volume set to Music as you do Effect in the general volume settings. Then, listen through the difficulty on normal playback rate and see if the hitsounds can be heard. If not, raise a problem in the “General (This difficulty)” section and inform the mapper. Additionally, find the correct volume for the mapper and add it to the suggestion. The easiest way to do this is by going to Timing and Pressing the Timing Setup Panel (or Pressing F6), and selecting the timestamp which is the first one behind the current timing point (e.g. selecting 00:43:392 - , if the sound is quiet at 00:43:495 - ,). From here, go to Audio, and adjust the volume until the hitsounds are audible enough, and suggest the new value to the mapper.
```

Comment:
Should maybe add a mention with recent rule change. Diffs with only insane and higher do not need hitsound additions.